index.html

```diff
@@ -0,0 +1,781 @@
+                </ul>
+            </li>
+            <li><a href="#">More…</a>
+                <ul class="sub-menu">
+                    <li><a href="http://www.nationalarchives.gov.uk/bookshop/">Bookshop</a></li>
+                    <li><a href="http://www.nationalarchives.gov.uk/imagelibrary/">Image library</a></li>
+                    <li><a href="http://www.nationalarchives.gov.uk/blog/">Blog</a></li>
+                    <li><a href="http://www.nationalarchives.gov.uk/media/">Podcasts and videos</a></li>
+                    <li><a href="http://www.nationalarchives.gov.uk/contact-us/">Contact us</a></li>
+                </ul>
+            </li>
+        </ul>
+    </div>
+</nav>
+
+<div id="breadcrumb-holder" class="tna-breadcrumb">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <div class="breadcrumbs"><span itemscope="" itemtype="http://data-vocabulary.org/Breadcrumb"><a
+                        href="http://www.nationalarchives.gov.uk/" itemprop="url"><span itemprop="title">Home</span></a></span>
+                    <span class="sep">&gt;</span>
+                    <span><a
+                            href="http://www.nationalarchives.gov.uk/help-with-your-research" itemprop="url"><span itemprop="title">Help with your research</span></a></span>
+                    <span class="sep">&gt;</span>
+                    <span><a
+                            href="http://www.nationalarchives.gov.uk/help-with-your-research/reading-old-documents/" itemprop="url"><span itemprop="title">Reading old documents</span></a></span>
+                    <span class="sep">&gt;</span>
+                    <span class="current"><span itemprop="title">Medieval Latin Lessons</span></span>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<main id="primary" role="main" class="content-area">
+
+
+    <div class="level-one">
+        <div class="container">
+            <div class="row" role="banner">
+                <div class="col-md-12">
+                    <article class="banner feature-img feature-img-bg"
+                             style="background-image: url(./img/banner.jpg);">
+                        <div class="entry-header">
+                            <h1>Medieval Latin Lessons</h1>
+                        </div>
+                    </article>
+                </div>
+            </div>
+
+        </div>
+    </div>
+
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <article>
+                    <div class="entry-header">
+                        <h2>Learn 1086 - 1733 Latin</h2>
+                    </div>
+                    <div class="row entry-content">
+                        <div class="col-xs-12 col-sm-8 col-md-8">
+                            <p><strong>Jump to:</strong> <a href="#stage-1">stage 1</a> | <a href="#stage-2">stage 2</a></p>
```

Comment:
Are these meant to be visible for both desktop and mobile?

Reply:
We will address this as content and not the template development